### DISCUSSION TITLE
Subak Climate Data Pledge

### DISCUSSION THREAD

**Discussion by rufuspollock** [datasets/commons#337] [Is It Open Data?]
**Summary: their definition of open data is not consistent with Open Definition due to being non-commercial use only**

Amazing to come across https://climatesubak.org/ and their work to get more Climate Data shared. However, their definition of Open Data is misleading as it defines open data as free for non-commercial use only.

## Context

A major focus of Subak is **data**. And it is supposedly it is all about **open data** 🙌 🍾 

### https://climatesubak.org/

> Share the data, save the planet. 

![image](https://user-images.githubusercontent.com/180658/132330272-b3a8d477-53f4-42da-a4b7-56ff6f731b98.png)

[Ed: I like bold claims for the power of data but this might be a bit far 😉. Anyway let's keep going ...]

### https://climatesubak.org/the-data-cooperative

![image](https://user-images.githubusercontent.com/180658/132329923-2f29f7a1-050c-4e3e-99d5-8642ec3833a3.png)

And they really emphasize it: https://climatesubak.org/about-us

> Our mission is to save the planet with shared data. We believe that creating a supportive ecosystem around climate non-profits will lead to faster and better climate action.  

![image](https://user-images.githubusercontent.com/180658/132330081-3e83b2e9-c80b-470d-9b20-a2775e4559c9.png)

This sounds *amazing*. Let's find out what they mean by open data. Let's read the pledge ...

### https://climatesubak.org/the-data-cooperative/#pledge

> OUR COMMITMENT TO OPEN DATA

But unfortunately it's not open because item 2 states:

> 2. Free (for non commercial use)
>
> We pledge that any data we uncover will be free for non-commercial use.

The [definition of Open Data](https://opendefinition.org/) requires that data be freely available for anyone to use for any purpose - this is absolutely central to the definition.

Aside: the pledge also include quite a few other commitments that whilst worthy I suspect will become burdensome/aspirational (based on extensive past experience) e.g. things like timeliness.

![image](https://user-images.githubusercontent.com/180658/132330575-28933072-203a-47d5-8714-f7d14d660776.png)

**Comment by agentGav** [comment]
I'd not actually seen this, so good catch.  

Almost every commercial meeting I'm in these days either confuses Open Data with Open Access (ie. if it has an openAPI somehow it's magically 'open') or fundamentally misunderstands how licensing works. I'll bring this up in my next chat with the Subak team :)  

We're expanding on the Data Spectrum 'Shared Data' element here https://icebreakerone.org/open-shared-closed/

**Reply by rufuspollock** [reply]
@agentGav 👍 did you get anywhere with the Subak team on this?

I notice that lots of url have changed and they now have their own spiffy data catalog https://data.subak.org (using ckan no less!)

**Reply by agentGav** [reply]
yes, there's an emerging collaboration between http://opennetzero.org (IB1) and the subak team (whose team we've worked with previously!)  - https://icebreakerone.org/icebreaker-principles might also be of interest...

**Reply by rufuspollock** [reply]
@agentGav so they clarified the principles to make clear this wasn't open? (or removed commercial restriction clause)?

**Reply by agentGav** [reply]
https://data.subak.org/ has the FAQ - I think the "pledge" has been removed from the site now (I can't find it)
